screens/Menu/AboutUs.js

```diff
@@ -0,0 +1,187 @@
+import React from "react";
+import { View, Text, StyleSheet, SafeAreaView, SectionList } from "react-native";
+import { ScrollView } from "react-native-gesture-handler";
+import { Paragraph } from "react-native-paper";
+import { Button } from "native-base";
+import { Feather } from "@expo/vector-icons";
+import PropTypes from "prop-types";
+
+/**
+ * Description: This is the about page for an
+ * informative summary about the application 
+ * and the info on the team memebers that have 
+ * participation of this application. Github usernames included.
+ */
+
+export const renderSeparator = () => {
```

Comment:
We seem to reuse this `renderSeparator` everywhere, maybe we should create a component so we can reduce code duplication!

Reply:
already said so